cves/kernel/CVE-2015-4178.yml

```diff
@@ -253,8 +257,8 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: The issue was with an unprivileged user performing a umount, not with i18n.
```

Comment:
"an" umount

Reply:
The "u" in "umount" makes a consonant sound, so it should be "a".